src/pages/faq.hbs

```diff
@@ -0,0 +1,47 @@
+{{#extend "raha/layout/base/layout"}}
+  {{#content "styles" mode="append"}}
+    .faq {
+      margin-top: 30px;
+      margin-bottom: 30px;
+    }
+
+    .anchorOffset {
+      display: block;
+      position: relative;
+      top: -80px;
+      visibility: hidden;
+    }
+    
+    .bareLink {
+      text-decoration: none;
+    }
+  {{/content}}
+
+  {{#content "body"}}
+    <main class="faq">
+      <article>
+        <h1 class="anchorOffset" id="top" data-t>faqTitle</h1>
+        
+        {{#each faq as |sectionValue sectionKey|}}
+            <h3>{{sectionKey}}</h3>
+            <ul>
+                {{#each this}}
+                    <li><a href="#{{@key}}" class="bareLink" data-t>faq.{{sectionKey}}.{{@key}}.q</a></li>
+                {{/each}}
+            </ul>
+        {{/each}}
+
+        {{#each faq as |sectionValue sectionKey|}}
+            <h1>{{sectionKey}}</h1>
+            {{#each this}}
+                <a id="{{@key}}" href="#" class="anchorOffset"></a>
+                <h3 data-t>faq.{{sectionKey}}.{{@key}}.q</h3>
+                <p data-t>faq.{{sectionKey}}.{{@key}}.a</p> 
+                <a href="#top" class="bareLink">&#10514;</a>
```

Comment:
That's kinda funky lol. But I assume that's like an arrow or something to go to top? If so a comment would be nice